node/clients/sellers.ts

```diff
@@ -19,7 +39,76 @@ export default class SellersClient extends JanusClient {
     })
   }
 
-  public async getSellers(): Promise<{ items: Seller[] }> {
-    return this.http.get(SELLERS_PATH)
+  public async getSellers(opts?: GetSellersOpts): Promise<GetSellersResponse> {
```

Comment:
Can we keep the original `getSellers` implementation and create a new `getSellersPaginated`? The `getSellers` original implementation is very simple and small, is not like we are duplicating a lot of code. This way we keep the original behavior without any changes.